Makefile

```diff
@@ -1,13 +1,18 @@
-sources = isrgrootx1.signing_policy.txt \
-          letsencryptauthorityx3.signing_policy.txt \
-          letsencryptauthorityx4.signing_policy.txt
+sources = isrgrootx1.signing_policy \
```

Comment:
Dropping the `.txt` suffix means these sources will be deleted by the `make clean` rule below.

Reply:
Fixed with the updated `make clean` in https://github.com/cilogon/letsencrypt-certificates/pull/3/commits/c40bc6cbd76460bfc2d5f3894cac731e457b1051